static/css/components/jquery.autocomplete.less

```diff
@@ -47,7 +47,7 @@
       line-height: normal;
 
       // stylelint-disable-next-line max-nesting-depth
-      &.ac_over {
+      &:hover {
         background-color: @light-yellow;
```

Comment:
This isn't working on staging; I think `.ac_results` used to _contain_ `ul`, but now I think `ul.ac_results` instead.

-- Confirmed this is the case.

Reply:
fixed.